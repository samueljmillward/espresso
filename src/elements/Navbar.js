import React, { useState } from "react";
import styled from "styled-components";

const Navbar = () => {
  return (
    <Nav>
      <h1>Navbar</h1>
    </Nav>
  );
};

export default Navbar;

const Nav = styled.div``;
