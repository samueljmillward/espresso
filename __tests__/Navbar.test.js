import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../src/elements/Navbar';

describe('renders UI', () => {
  it('Navbar', () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
  });
});
