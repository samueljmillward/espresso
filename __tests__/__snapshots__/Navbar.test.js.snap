// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI Navbar 1`] = `
<styled.div>
  <styled.a
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "innerRef": [Function],
          "onClick": [Function],
          "replace": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      }
    }
    href="/"
  >
    Es
    <span>
      presso
    </span>
  </styled.a>
  <styled.div
    onClick={[Function]}
  >
    <span />
    <span />
    <span />
  </styled.div>
  <styled.div
    isOpen={false}
  >
    <styled.a
      href="/beans"
    >
      My Beans
    </styled.a>
    <styled.a
      href="/brews"
    >
      Pull A Shot
    </styled.a>
  </styled.div>
</styled.div>
`;
